src/library-authoring/author-library/specs/LibraryAuthoringPage.spec.jsx

```diff
@@ -136,51 +136,52 @@ testSuite('<LibraryAuthoringPageContainer />', () => {
     expect(fetchLibraryBlockView.fn).toHaveBeenCalledTimes(1);
   });
 
-  it('Fetches block LTI URL to clipboard', async () => {
-    const library = libraryFactory({ allow_lti: true });
-    const blocks = makeN(blockFactoryLine([], { library }), 2);
+  // todo: figure out how LTI stuff works
```

Comment:
@arbrandes i'm not sure how to test this. i'll gladly bring back the buttons and uncomment these tests once i know how to set up a library to have LTI stuff

Reply:
Let's push the LTI stuff back until the rest is done - maybe even further.  It requires backend fixes before we can revisit the frontend properly.